app/src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters.kt

```diff
@@ -53,45 +51,6 @@ fun setImageDrawable(imageView: ImageView, thumbnailGraphic: LessonThumbnailGrap
   )
 }
 
-/**
- * Binds the specified [SkillThumbnailGraphic] as the source for the [ImageView]. The view should be specified to have
- * no width/height (when sized in a constraint layout), and use centerCrop for the image to appear correctly.
- */
-@BindingAdapter("android:src")
-fun setImageDrawable(imageView: ImageView, thumbnailGraphic: SkillThumbnailGraphic) {
-  setImageDrawable(
-    imageView, when (thumbnailGraphic) {
-      SkillThumbnailGraphic.IDENTIFYING_THE_PARTS_OF_A_FRACTION -> R.drawable.topic_fractions_01
-      SkillThumbnailGraphic.WRITING_FRACTIONS -> R.drawable.topic_fractions_02
-      SkillThumbnailGraphic.EQUIVALENT_FRACTIONS -> R.drawable.topic_fractions_03
-      SkillThumbnailGraphic.MIXED_NUMBERS_AND_IMPROPER_FRACTIONS -> R.drawable.topic_fractions_04
-      SkillThumbnailGraphic.COMPARING_FRACTIONS -> R.drawable.topic_fractions_05
-      SkillThumbnailGraphic.ADDING_AND_SUBTRACTING_FRACTIONS -> R.drawable.topic_fractions_06
-      SkillThumbnailGraphic.MULTIPLYING_FRACTIONS -> R.drawable.topic_fractions_07
-      SkillThumbnailGraphic.DIVIDING_FRACTIONS -> R.drawable.topic_fractions_08
-      SkillThumbnailGraphic.DERIVE_A_RATIO -> R.drawable.topic_ratios_01
-      else -> R.drawable.topic_fractions_01
-    }
-  )
-}
-
-/**
- * Binds the specified [SkillThumbnailGraphic] as the source for the [ImageView]. The view should be specified to have
- * no width/height (when sized in a constraint layout), and use centerCrop for the image to appear correctly.
- */
-@BindingAdapter("android:src")
-fun setImageDrawable(imageView: ImageView, thumbnailGraphic: SubtopicThumbnailGraphic) {
-  setImageDrawable(
-    imageView, when (thumbnailGraphic) {
-      SubtopicThumbnailGraphic.WHAT_IS_A_FRACTION -> R.drawable.topic_fractions_01
```

Comment:
These cannot be removed and as a result of this all images in Fractions - Revision list of not visible. Basically you will need to update the `setImageDrawable ` in line 39 to fix this.
It should like this: 
![Screenshot_1586944655](https://user-images.githubusercontent.com/9396084/79324838-d579b300-7f2d-11ea-8247-a5be5f9e4eca.png)

Reply:
Done :)